src/App.tsx

```diff
@@ -2,166 +1,105 @@
-
-                  <td>m</td>
-                  <td>1761</td>
-                  <td>1833</td>
-                  <td>-</td>
-
-                  <td>
-                    <a href="#/people/jacobus-bernardus-van-brussel-1736">
-                      Jacobus Bernardus van Brussel
-                    </a>
-                  </td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/lievijne-jans-1542"
-                    >
-                      Lievijne Jans
-                    </a>
-                  </td>
-
-                  <td>f</td>
-                  <td>1542</td>
-                  <td>1582</td>
-                  <td>-</td>
-                  <td>-</td>
-                </tr>
-
-                <tr data-cy="person">
-                  <td>
-                    <a href="#/people/bernardus-de-causmaecker-1721">
-                      Bernardus de Causmaecker
-                    </a>
-                  </td>
-
-                  <td>m</td>
-                  <td>1721</td>
-                  <td>1789</td>
-
-                  <td>
-                    <a
-                      className="has-text-danger"
-                      href="#/people/livina-haverbeke-1692"
-                    >
-                      Livina Haverbeke
-                    </a>
-                  </td>
-
-                  <td>
-                    <a href="#/people/lieven-de-causmaecker-1696">
-                      Lieven de Causmaecker
-                    </a>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
+              Home
+            </Link>
+
+            <Link
+              className={cN('navbar-item', {
+                'has-background-grey-lighter': location.startsWith('/people'),
+              })}
+              to="/people"
+            >
+              People
+            </Link>
           </div>
         </div>
-      </div>
-    </main>
-  </div>
-);
+      </nav>
+
+      <main className="section">
+        <div className="container">
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/people">
+              <Route
+                index
+                element={
+                  <PeopleTable
+                    people={people}
+                    isLoading={isLoading}
+                    hasError={hasError}
+                  />
+                }
+              />
+              <Route
+                path=":selectedUserSlug"
+                element={
+                  <PeopleTable
+                    people={people}
+                    isLoading={isLoading}
+                    hasError={hasError}
+                  />
+                }
+              />
+            </Route>
```

Comment:
The nested `Route` components inside the `/people` route could be simplified by using a `Routes` component directly inside the `PeopleTable` component if it handles different views based on the route. This would make the routing logic cleaner.